### PR TITLE
Porting a first part of the og.base tests to integrationtests.

### PR DESCRIPTION
**Pull request by phgross**
Should help to shorten the test duration.

### CODE REVIEW

File: opengever/testing/integration_test_case.py [hunk 1]
```diff
@@ -491,3 +493,9 @@ def get_allowed_roles_and_users_for(self, obj):
         catalog = api.portal.get_tool('portal_catalog')
         rid = catalog.getrid('/'.join(obj.getPhysicalPath()))
         return catalog.getIndexDataForRID(rid).get('allowedRolesAndUsers')
+
+    def assert_journal_entry(self, obj, type, title, entry=-1):
```
Comment (deiferni) [inline opengever/testing/integration_test_case.py]: this redefines built-in `type`, please rename.

Comment (deiferni) [inline opengever/testing/integration_test_case.py]: `entry` -> `index`?

File: opengever/testing/integration_test_case.py [hunk 2]
```diff
@@ -499,3 +499,7 @@ def assert_journal_entry(self, obj, type, title, entry=-1):
         action = entry.get('action')
         self.assertEquals(type, action.get('type'))
         self.assertEquals(title, translate(action.get('title')))
+
+    def get_selected_paths(self, *objects):
```
Comment (deiferni) [inline opengever/testing/integration_test_case.py]: `make_path_param`?

**Comment by phgross** [comment]
@deiferni updated ...

**Comment by deiferni** [comment]
@phgross tests are failing 😢

**Comment by phgross** [comment]
@deiferni now they pass

**Pull request merged**